pkg/controllers/jobset_controller.go

```diff
@@ -152,6 +151,9 @@ func (r *JobSetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 // getChildJobs gets jobs owned by the JobSet then categorizes them by status (active, successful, failed).
 // Another list (`delete`) is also added which tracks jobs marked for deletion.
 func (r *JobSetReconciler) getChildJobs(ctx context.Context, js *jobset.JobSet) (*childJobs, error) {
+	log := ctrl.LoggerFrom(ctx).WithValues("jobset", klog.KObj(js))
```

Comment:
pass in the logger as a parameter instead of creating it every time, here and in all other places

Reply:
discussed via chat and decided on using `log := ctrl.LoggerFrom(ctx)` to retrieve logger from context, and removing the extraneous WithValues and `LoggerInto` calls since they are already done in the Reconcile() method.